src/plugins/legacy/itkDataImage/interactors/medVtkViewItkDataImageInteractor.cpp

```diff
@@ -277,7 +277,7 @@ void medVtkViewItkDataImageInteractor::initParameters(medAbstractImageData* data
     }
     else
     {
-       this->opacityParameter()->setValue(1);
+       this->opacityParameter()->setValue(0.99);// if 1.0 : no signal is since the initial is 1.0 but it has been set without a emiting signal
```

Comment:
I do not understand i think. Are you using 0.99 to force the emit of the signal? Will it be set to 1 after?